tests/unit/cli/test_stack.py

```diff
@@ -220,3 +226,59 @@ def test_deleting_stack_with_flag_succeeds(clean_repo) -> None:
     assert result.exit_code == 0
     with pytest.raises(KeyError):
         clean_repo.get_stack("arias_new_stack")
+
+
+def test_stack_export(clean_repo) -> None:
+    """Test exporting default stack succeeds."""
+    runner = CliRunner()
+    result = runner.invoke(export_stack, ["default.yaml", "default"])
+    assert result.exit_code == 0
+    assert os.path.exists("default.yaml")
+
+
+def test_stack_export_delete_import(clean_repo) -> None:
+    """Test exporting, deleting, then importing a stack succeeds."""
+    # create new stack
+    artifact_store_name = "arias_artifact_store"
```

Comment:
🐈‍⬛

Reply:
important to stick to the naming convention ;)

Reply:
🐈‍⬛